js/ui/components/polkitAgent.js

```diff
@@ -150,6 +153,16 @@ const AuthenticationDialog = new Lang.Class({
         this._passwordBox.add(this._passwordEntry,
                               { expand: true });
 
+        this._inputSourceManager = InputSourceManager.getInputSourceManager();
+        this._inputSourceIndicator = new InputSourceIndicator(this, false);
```

Comment:
The indicator already has a reference to the input source manager (https://github.com/endlessm/gnome-shell/blob/cf9b988ef5cd9483eeadbc929f1451d85424b2f3/js/ui/status/keyboard.js#L829).